db/seeds.rb

```diff
@@ -5,3 +5,38 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+item_contents = ([{ name: "Chocolate Toast", description: "Dark chocolate on toast",
+        price: "3.00"}, { name: "Cake Toast", description: "Lemon poppyseed cake on toast", price: "4.00"},
+        { name: "Brownie Toast", description: "Delicious dark chocolate brownie on toast", price: "4.00"},
```

Comment:
Line is too long. [106/80]<br>Align the elements of an array literal if they span more than one line.<br>Space inside } missing.